mondrian/src/main/java/mondrian/rolap/aggmatcher/ExplicitRecognizer.java

```diff
@@ -495,7 +495,7 @@ public int compare(
         Optional<Map.Entry<org.eclipse.daanse.rdb.structure.api.model.Column, org.eclipse.daanse.rdb.structure.api.model.Column>> op = measuresFactCount.entrySet()
         		.stream().filter(e -> e.getKey().getName().equals(measureName)).findAny();         
         String factCountColumnName;
-        if (op.isPresent() && Util.isEmpty(op.get().getValue().getName())) {
+        if (op.isPresent() && !Util.isEmpty(op.get().getValue().getName())) {
```

Comment:
How was this in history? Sure about that?

Reply:
`@Override
	protected String getFactCountColumnName
            (final JdbcSchema.Table.Column.Usage aggUsage) {
        String measureName = aggUsage.getColumn().getName();
        Map<String, String> measuresFactCount = tableDef.getMeasuresFactCount();
        String factCountColumnName = measuresFactCount.get(measureName);
        if (Util.isEmpty(factCountColumnName)) {
            factCountColumnName = getFactCountColumnName();
        }
        return factCountColumnName;
    }`